packages/client-sdk-nodejs/src/internal/cache-data-client.ts

```diff
@@ -139,6 +139,7 @@ import Equal = common.Equal;
 import NotEqual = common.NotEqual;
 import PresentAndNotEqual = common.PresentAndNotEqual;
 import AbsentOrEqual = common.AbsentOrEqual;
+import {TestRetryMetricsMiddleware} from '../../test/test-retry-metrics-middleware';
```

Comment:
This raises a pretty big red flag for me. Our SDK code shouldn't have any awareness of our tests. It looks like the intent here is to make sure the test middleware is last in the list. Is there a way to accomplish that by ordering them in the test config instead?